src/mcts/node.cc

```diff
@@ -237,7 +237,18 @@ void Node::MakeNotTerminal() {
   // If we have edges, we've been extended (1 visit), so include children too.
   if (edges_) {
     n_++;
-    for (const auto& child : Edges()) n_ += child.GetN();
+    for (const auto& child : Edges()) {
+      const auto n = child.GetN();
+      if (n > 0) {
+        n_ += n;
+        q_ -= child.GetQ(0.0f) * n;
```

Comment:
Would be nice to add a comment why it is `-=` here instead of `+=`.

Reply:
Would `+= -child.GetQ()` instead be more immediately clear without a comment?

Reply:
Switched to `+=` and added comment:
https://github.com/LeelaChessZero/lc0/blob/b9b5e2fe8ceb9b4aa3abebad8f037547f3e9905c/src/mcts/node.cc#L244-L245